bodhi/server/services/updates.py

```diff
@@ -448,6 +450,15 @@ def new_update(request):
     # it since the models don't care about a csrf argument.
     data.pop('csrf_token')
 
+    # Require severity level for a 'security' update
+    if data['type'] == UpdateType.security and data['severity'] not in (UpdateSeverity.low,
+                                                                        UpdateSeverity.medium,
+                                                                        UpdateSeverity.high,
+                                                                        UpdateSeverity.urgent):
+        log.warn('No severity specified for a security update.')
+        request.errors.add('body', 'severity', 'Must specify severity for a security update')
+        return
```

Comment:
Let's use the validators to require this instead of doing it inline here. The purpose of the validators is to make sure the request is valid, which is the goal here too.